packages/perseus/src/widgets/__tests__/input-number.test.ts

```diff
@@ -337,18 +337,4 @@ describe("focus state", () => {
         // Assert
         expect(gotFocus).toBeTrue();
     });
-
```

Comment:
These are no longer required now that we're calling the interactionCallback in the server-item-renderer.